featuretools/computational_backends/pandas_backend.py

```diff
@@ -518,17 +521,21 @@ def wrap(df):
             args = [df[v] for v in variable_ids]
 
             if f.uses_calc_time:
-                d[f.get_name()] = func(*args, time=time_last)
+                values = func(*args, time=time_last)
             else:
-                d[f.get_name()] = func(*args)
+                values = func(*args)
 
+            if f.number_output_features > 1:
+                names = f.output_feature_names()
+                d.update({name: None for name in names})
```

Comment:
why is this needed? seems like every entry in names will get add to dict 2 lines below